assets/javascripts/kitten/components/form/rich-text-input.js

```diff
@@ -18,6 +18,7 @@ export class RichTextInput extends React.Component {
 
   handleChange(value) {
     this.setState({ content: value })
+    this.props.onChange({ value })
```

Comment:
Tu as besoin d'un objet quand tu le récupères après ou simplement la `value` ?
`this.props.onChange(value)` est peut être plus simple, à voir…